### PR TITLE
Redcore 515 compatibility with joomla 3 7

### PR DESCRIPTION
**Pull request by Kixo**

### CODE REVIEW

File: extensions/plugins/system/redcore/redcore.xml [hunk 1]
```diff
@@ -7,7 +7,7 @@
     <license>GNU General Public License version 2 or later; see LICENSE.txt</license>
     <authorEmail>email@redcomponent.com</authorEmail>
     <authorUrl>www.redcomponent.com</authorUrl>
-    <version>1.9.0</version>
+    <version>1.9.1</version>
```
Comment (jatitoam) [inline extensions/plugins/system/redcore/redcore.xml]: Probably 1.10.0?

**Comment by puneet0191** [comment]
is very strange failure, not exactly sure of the reason, but I think we just need to update Joomla-Browser to 3.7.0 Release tag (I plan to do it today EOD), Update RoboFile.ini to 3.7.0 Joomla version, I am not sure if we need changes in redCOMPONENT Robo.

**Comment by puneet0191** [comment]
I think this one worked, but failing on php 5.5

**Comment by Kixo** [comment]
yes, I managed to pin it down :) and yes, some dependencies require PHP to be at least 5.6 or they cannot even be downloaded, so I am taking PHP 5.5 out of tests here

**Pull request merged**